src/js/tech/tech.js

```diff
@@ -437,6 +437,15 @@ class Tech extends Component {
     return '';
   }
 
+  static isTech(component) {
+    /* jshint proto:true */
+    let proto = component.__proto__;
+    /* jshint proto:false */
+    if (Object.getPrototypeOf) {
```

Comment:
I guess I should change that. Also, should it support instances of techs as well?